channeld/channeld.c

```diff
@@ -1798,6 +1799,19 @@ static void handle_peer_revoke_and_ack(struct peer *peer, const u8 *msg)
 				 "Unexpected revoke_and_ack");
 	}
 
+	/* Submit the old revocation secret to the signer so it can
+	 * independently verify it is correct. It is also validated
```

Comment:
maybe give a more specific motivation like "so it can verify that the latest state is committed"